src/screens/RecipeScreen.tsx

```diff
@@ -11,19 +11,43 @@ const _screen = Dimensions.get("screen");
 
 export default function RecipeScreen({ route }: { route: any }) {
   const { recipeId } = route.params;
-  const [recipeInfos, setRecipeInfos] = React.useState<RecipeScreenInfo>({title: "", summary: "",});
+  const [recipeInfos, setRecipeInfos] = React.useState<RecipeScreenInfo>({
```

Comment:
In the case where the API call fails and returns an empty array (it couldn't process our request), we need to handle this. Can we allow for this state variable to be a RecipeScreenInfo object AND undefined, or empty object, etc.?